src/firefly/js/visualize/ui/FileUploadViewPanel.jsx

```diff
@@ -173,9 +191,9 @@ export function FileUploadViewPanel({setSubmitText, acceptMoc}) {
                                                        }}/> }
                         </div>
                     </div>
-                    <FileAnalysis {...{report, summaryModel, detailsModel,tablesOnly, isMoc, UNKNOWN_FORMAT, acceptMoc}}/>
-                    <ImageDisplayOption summaryTblId={summaryTblId} currentReport={report} currentSummaryModel={summaryModel}/>
-                    <TableDisplayOption {...{isMoc, isDatalink, summaryTblId, currentReport:report, currentSummaryModel:summaryModel, acceptMoc}}/>
+                    <FileAnalysis {...{report, summaryModel, detailsModel, isMoc, UNKNOWN_FORMAT, acceptList}}/>
+                    <ImageDisplayOption summaryTblId={summaryTblId} currentReport={report} currentSummaryModel={summaryModel} acceptList={acceptList}/>
```

Comment:
you might want to make this component to use the object style of passing props just to keep in more consistent with the line before and after.